unet/performance_evaluation.py

```diff
@@ -0,0 +1,236 @@
+"""
```

Comment:
Why are you adding this?

Reply:
I guess I should'nt have integrated the unet folder within my branch, I thought I can have it here for reference and clean up later, some of the test files you have flagged are from the unet tests.. I will remove this from the branch